configtx/config.go

```diff
@@ -78,28 +78,41 @@ type standardConfigPolicy struct {
 // ConfigTx wraps a config transaction.
 type ConfigTx struct {
 	// original state of the config
-	original *cb.Config
+	original *OriginalConfig
```

Comment:
Original would be referenced as `*configtx.Original` and Updated would be `*configtx.Updated`. Seems okay?

Thoughts?